src/views/profile/edit.ejs

```diff
@@ -114,19 +114,19 @@
         <option value="CL">Chile</option>
         <option value="CN">China</option>
         <option value="CX">Christmas Island</option>
-        <option value="CC">Cocos (Keeling) Islands</option>
+        <option value="CC">Cocos Islands</option>
         <option value="CO">Colombia</option>
         <option value="KM">Comoros</option>
         <option value="CG">Congo</option>
-        <option value="CD">Congo, the Democratic Republic of the</option>
         <option value="CK">Cook Islands</option>
         <option value="CR">Costa Rica</option>
         <option value="CI">Côte d'Ivoire</option>
         <option value="HR">Croatia</option>
         <option value="CU">Cuba</option>
         <option value="CW">Curaçao</option>
         <option value="CY">Cyprus</option>
-        <option value="CZ">Czech Republic</option>
+        <option value="CZ">Czechia</option>
+        <option value="CD">Democratic Republic of the Congo</option>
```

Comment:
Should this just be Congo? Having "Democratic" at the front would change the alphabetic ordering and might make it harder to find?

Reply:
The full name of this country is Democratic Republic of the Congo, not to be confused with (Republic of the) Congo

Reply:
Ah interesting. Cool, let's keep it as is!